.github/workflows/ci.yml

```diff
@@ -11,8 +11,8 @@ jobs:
   determine_docker_org_and_tag:
     runs-on: ubuntu-latest
     outputs:
-      docker_organization: ${{ steps.docker-org-and-tag.outputs.docker_organization }}
-      docker_image_tag: ${{ steps.docker-org-and-tag.outputs.docker_image_tag }}
+      docker_organization: usdotfhwastolcandidate
+      docker_image_tag: prowler
```

Comment:
This should be unnecessary, since from CI steps, it should already detect these docker_organization and docker_image_tag

Reply:
updated